core/src/main/java/com/yahoo/imapnio/async/internal/ImapAsyncSessionImpl.java

```diff
@@ -208,10 +229,7 @@ private void sendRequest(@Nonnull final Object request) throws ImapAsyncClientEx
         }
 
         final ImapCommandEntry entry = requestsQueue.peek();
-        if (logger.isDebugEnabled()) {
-            logger.debug(CLIENT_LOG_REC, sessionId, command.getLogLine());
-        }
-        sendRequest(entry.getRequest().getTerminateCommandLine());
+        sendRequest(entry.getRequest().getTerminateCommandLine(), false);
```

Comment:
why false?

Reply:
Fixed.  I was thinking only IdleCommand has terminateCommandLine and it is data in-sensitive, so trying to hard-code it.